Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -28,10 +29,50 @@
 
 namespace ROS2
 {
+    bool CollidersMaker::GuessIsWheel(const AZStd::string& entityName)
+    {
+        AZStd::regex wheel_regex("(?i)wheel|(?i)wh_|(?i)(_wh)");
```

Comment:
Has (case-insensitive) "wheel" anywhere in name, but has no "joint" anywhere.

Reply:
has a collider
has a visual - in the link (URDF): visuals in entity/prefab are in sub-entity
has a continuous joint (component already)
and the name